consul-lambda-registrator/trigger.go

```diff
@@ -345,12 +355,12 @@ func (e Environment) constructUpsertEvents(lambdas map[*EnterpriseMeta]map[strin
 
 // constructUpsertEvents determines which delete events need to be processed to
 // synchronize Consul with Lambda.
-func (e Environment) constructDeleteEvents(lambdas map[*EnterpriseMeta]map[string]Event, consulServices map[*EnterpriseMeta]map[string]struct{}) []Event {
+func (e Environment) constructDeleteEvents(lambdas eventMap, consulServices serviceMap) []Event {
```

Comment:
Yay for naming things making the code significantly easier to read

Reply:
Thanks.